### DISCUSSION TITLE
Does my SO VM need to be powered on before I use its management IP?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5651] [Unsupported Versions] [answered]
Hi, 
    I could try this, but i want to know WHY this would or would not work? 
 thanks

**Comment by clairmont32** [comment, marked as answer]
Can you clarify what you're attempting to do and what on hardware? Are you talking about something like IPMI or iDrac access?

**Reply by iqworks** [reply]
sure, ..... i am using windows 10 on my host laptop. I currently have SO import installed at 4GB RAM. 
I am wondering if I use that SO import management IP in a browser with this SO VM powered on, or using the IP with this SO VM powered off. 
Not sure what "IPMI or iDrac access" are, so i have to look that up. But maybe one of these two would allow you to access the SO VM whether they are powered on or off?? 
OR, maybe it depends on if you are using a browser from within another VM or your host browser?

**Reply by clairmont32** [reply]
Yes, you will need the VM powered on to access anything within SO. You should hit the management IP a couple minutes after everything has started up.

IPMI/iDrac are commercial methods of accessing a hardware appliance for basic management and troubleshooting prior to the server being turned on. They wont apply to your current setup but I used them as an example to see if it was something along the lines of what you were looking to perform.
